### PR TITLE
fix #440 only get these tables information in custom the option ‘-tab…

### LINKED ISSUE #440: always get all tables information in the db,when you custom the command 'api' option '-tables'
When  many tables in the db, get all tables information will cost a lot of time,the custom option '-tables' is unused.

### PR DESCRIPTION
**Pull request by hudangwei**
…les’

### CODE REVIEW

File: generate/g_appcode.go [hunk 1]
```diff
@@ -182,6 +182,7 @@ type OrmTag struct {
 	RelFk       bool
 	ReverseMany bool
 	RelM2M      bool
+	Comment     string //字段注解
```
Comment (astaxie) [inline generate/g_appcode.go]: please change this comment to english

Comment (hudangwei) [inline generate/g_appcode.go]: it's done.

File: generate/g_appcode.go [hunk 2]
```diff
@@ -255,7 +256,7 @@ func (tag *OrmTag) String() string {
 	if len(ormOptions) == 0 {
 		return ""
 	}
-	return fmt.Sprintf("`orm:\"%s\"`", strings.Join(ormOptions, ";"))
+	return fmt.Sprintf("`orm:\"%s\" description:\"%s\"`", strings.Join(ormOptions, ";"), tag.Comment)
```
Comment (astaxie) [inline generate/g_appcode.go]: if tag.Comment is empty it will create the empty descriptio. Why you didn't follow the other logic, like Default.

Comment (hudangwei) [inline generate/g_appcode.go]: i have changed the logic.

File: generate/g_appcode.go [hunk 3]
```diff
@@ -255,6 +256,9 @@ func (tag *OrmTag) String() string {
 	if len(ormOptions) == 0 {
 		return ""
 	}
+	if tag.Comment != "" {
+		return fmt.Sprintf("`orm:\"%s\" description:\"%s\"`", strings.Join(ormOptions, ";"), tag.Comment)
```
Comment (astaxie) [inline generate/g_appcode.go]: `strings.Join(ormOptions, ";")` use `;` as the sperator. but you are using hard code here and alsp use space. please read the code first.

Comment (hudangwei) [inline generate/g_appcode.go]: description is for setting the swagger.Schema.Description.
https://github.com/beego/bee/blob/master/generate/swaggergen/g_docs.go#L988

**Pull request merged**